src/python/tests/core/datastore/data_handler_test.py

```diff
@@ -681,3 +681,41 @@ def test_blackbox_fuzzer_testcase_with_default_help_format(self):
          'ASAN_OPTIONS="handle_abort=1:redzone=512" '
          'UBSAN_OPTIONS="halt_on_error=1" ./binary'
         ).format(id=testcase.key.id()))
+
+  def test_blaze_test_args(self):
+    """Test the function with a blackbox fuzzer test case, with HELP_FORMAT
```

Comment:
nit: the comment doesn't seem fully correct, it's not "blackbox" case

Reply:
Done, good catch!